tools/cli/internal/openapi/oasdiff_test.go

```diff
@@ -1142,3 +1145,198 @@ func TestUpdateExternalRefReqBody(t *testing.T) {
 		})
 	}
 }
+
+func TestHandlePathConflict(t *testing.T) {
+	testCases := []struct {
+		name          string
+		basePath      *openapi3.PathItem
+		basePathName  string
+		specDiff      *diff.Diff
+		expectedError error
+	}{
+		{
+			name: "No Conflict - Identical Paths",
+			basePath: &openapi3.PathItem{
+				Get: &openapi3.Operation{
+					Extensions: map[string]interface{}{
+						"x-xgen-soa-migration": map[string]interface{}{
+							"allowDocsDiff": "false",
+						},
+					},
+				},
+			},
+			basePathName: "/test",
+			specDiff: &diff.Diff{
+				PathsDiff: &diff.PathsDiff{
+					Modified: map[string]*diff.PathDiff{},
+				},
+			},
+			expectedError: nil,
+		},
+		{
+			name: "Conflict with AllowDocsDiff",
+			basePath: &openapi3.PathItem{
+				Get: &openapi3.Operation{
+					Extensions: map[string]interface{}{
+						"x-xgen-soa-migration": map[string]interface{}{
+							"allowDocsDiff": "true",
```

Comment:
if we have  `AllowDocsDiffNotSupportedError`, why allowDocsDiff can be set to true?

Reply:
I think answer is the same as it's a separate ticket to support it (cause initially we were not going to support)